app/models/user.rb

```diff
@@ -197,6 +197,21 @@ def weekly_note_tally(span = 52)
     weeks
   end
 
+  def daily_note_tally(span = 365)
+      days = {}
+      (0..span).each do |day|
+          time=Time.now.in_time_zone(0).beginning_of_day.to_i
+          days[(time-day.days.to_i)] = Node.select(:created)
+          .where(uid: drupal_user.uid, 
+                                        type: 'note',
```

Comment:
Also here, do you think we could make the indentation really consistent? Not such a big deal but it really helps us keep the code tidy long-term. Thank you!

Reply:
Sure @jywarren I would do that